tests/meta/backup/test_meta_backup.py

```diff
@@ -55,7 +53,7 @@ def test_maximum_processing_queue_not_full(mock_store):
     # WHEN there are no flow cells being retrieved from PDC
     mock_store.get_flow_cells_by_statuses(
         flow_cell_statuses=[FlowCellStatus.PROCESSING]
```

Comment:
Do we need to pass the `flow_cell_statuses` argument?

Reply:
It works without, I removed it.